src/foam/u2/view/LoginView.js

```diff
@@ -193,7 +193,7 @@ foam.CLASS({
       factory: function() {
         return !! this.model.backLink_ ?
           this.model.backLink_ :
-          !! this.appConfig.url ? this.appConfig.url : undefined;
+          !! this.appConfig.externalUrl ? this.appConfig.externalUrl : undefined;
```

Comment:
does this not work without the double negation !! ?  What bug are you protecting against with this?

Reply:
> does this not work without the double negation !! ? What bug are you protecting against with this?

I'm not sure. I did not write those code. I only changed url to externalUrl. I can rewrite the logic if you suggest.

Reply:
You could also just do: this.appConfig.externalUrl || undefined;

Reply:
re: rewrite - yes please.  Update to Kevin's recommendation.  Thanks.